src/app/dataset-view/dataset.component.ts

```diff
@@ -89,6 +96,12 @@ export class DatasetComponent
                     this.searchValue = value;
                 },
             ),
+            this.activatedRoute.queryParams.subscribe((params: Params) => {
+                if (params) {
+                    this.datasetViewType = params.tab;
```

Comment:
What happens if URL contains a tab that we do not know?

Reply:
I removed this activatedRoute in new commit.